### PR TITLE
feat(FOROME-1399): Added animation for cards

### PR DESCRIPTION
**Pull request by AndreyUstyumenko**
https://quantori.atlassian.net/browse/FOROME-1399

### CODE REVIEW

File: src/ui/card/card.tsx [hunk 1]
```diff
@@ -1,19 +1,73 @@
...
+    const cardRef = useRef<HTMLDivElement>(null)
+
+    useEffect(() => {
+      window.setTimeout(() => setIsMounted(true), 0)
+    }, [])
+
+    const renderCard = (state = '') => {
+      return (
+        <div
+          ref={innerRef ? innerRef : cardRef}
```
Comment (QSdmitrioul) [inline src/ui/card/card.tsx]: Why don't use `useForkRef `?

File: src/ui/card/card.tsx [hunk 2]
```diff
@@ -1,19 +1,73 @@
...
+  Ref,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+import { Transition } from 'react-transition-group'
 import cn, { Argument } from 'classnames'
 
-interface ICardProps {
+interface CardProps {
```
Comment (Lapk1n) [inline src/ui/card/card.tsx]: why not ICardProps?

File: src/ui/card/card.tsx [hunk 3]
```diff
@@ -1,19 +1,73 @@
...
+  }: CardProps): ReactElement => {
+    const [isMounted, setIsMounted] = useState(false)
+    const cardRef = useRef<HTMLDivElement>(null)
+
+    useEffect(() => {
+      window.setTimeout(() => setIsMounted(true), 0)
+    }, [])
+
+    const renderCard = (state = '') => {
+      return (
```
Comment (Lapk1n) [inline src/ui/card/card.tsx]: will look a bit better without return(...)

File: src/ui/card/card.tsx [hunk 4]
```diff
@@ -1,19 +1,73 @@
...
+        <div
+          ref={innerRef ? innerRef : cardRef}
+          className={cn(
+            'shadow-card p-4 rounded-lg',
```
Comment (Lapk1n) [inline src/ui/card/card.tsx]: p-4 shadow-card rounded-lg

File: src/ui/card/card.tsx [hunk 5]
```diff
@@ -1,19 +1,71 @@
...
+
+    useEffect(() => {
+      window.setTimeout(() => setIsMounted(true), 0)
+    }, [])
+
+    const renderCard = (state = '') => (
+      <div
+        ref={innerRef ? innerRef : cardRef}
+        className={cn(
+          'p-4 shadow-card rounded-lg',
```
Comment (QSdmitrioul) [inline src/ui/card/card.tsx]: Styles.card instead of

**Review by Lapk1n — APPROVED** [review]

**Review by QSdmitrioul — APPROVED** [review]

**Pull request merged**